tests/_internal/test_client_factories.py

```diff
@@ -148,10 +151,11 @@ def test_get_departing_employee_client_calls_get_stored_value_with_expected_key(
         factory = MicroserviceClientFactory(
             TEST_ROOT_URL, mock_session, session_factory, user_context, key_value_store_client
         )
-        factory.get_departing_employee_client()
-        key_value_store_client.get_stored_value.assert_called_once_with(
-            "employeecasemanagement-API_URL"
-        )
+        user_client = UserClient(mock_session)
```

Comment:
Per my above comment, switch this test back to check that the key value store client is only called once after we make the change to use a single session for all detection list clients

Reply:
Done.